webapp/src/components/kanban/kanbanCard.tsx

```diff
@@ -115,6 +115,7 @@ const KanbanCard = (props: Props) => {
                                 false,
                                 'duplicate card',
                                 false,
+                                undefined,
```

Comment:
With my suggested changes you may need to pass an empty object `{}` here.

Reply:
Done